lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -214,6 +214,77 @@ speed:
   config_set_append: "speed %s"
   default_value: "auto"
 
+stp_bpdufilter:
+  kind: string
```

Comment:
Shouldn't the `kind:` field be set to `boolean` for `stp_bpdufilter:`, `stp_bpduguard:`, `stp_guard:`, `stp_port_type:`?